### PR TITLE
feat(auth-admin): Auth admin client patch

### PR DESCRIPTION
**Pull request by GunnlaugurG**
# ...

Attach a link to issue if relevant

## What

* Rest endpoint to patch client
* Graphql layer to patch client
* Connected frontend to the graphql layer

## Why

So clients can be edited by users

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx [hunk 1]
```diff
@@ -85,53 +75,81 @@ const AdvancedSettings = ({
...
-          value={`${consent}`}
+          defaultChecked={inputValues.requireConsent}
+          checked={inputValues.requireConsent}
+          onChange={(e) => {
+            setInputValues({
+              ...inputValues,
+              requireConsent: e.target.checked,
+            })
+          }}
+          value={'true'}
```
Comment (saevarma) [inline libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx:86]: ~When the value is just hardcoded string we can drop the brackets.
```suggestion
          value="true"
```

File: libs/portals/admin/ids-admin/src/components/Client/Translations.tsx [hunk 2]
```diff
@@ -39,10 +35,11 @@ const Translations = ({ translations }: TranslationsProps) => {
   return (
     <ContentCard
       title={formatMessage(m.translations)}
-      onSave={(saveOnAllEnvironments) => {
-        console.log('saveOnAllEnvironments: ', saveOnAllEnvironments)
+      onSave={() => {
```
Comment (saevarma) [inline libs/portals/admin/ids-admin/src/components/Client/Translations.tsx:38]: Looks like the `onSave` function is redundant, and in the other components. What is the purpose of the function other than control rendering of the 'Save in all environments' checkbox?

**Review by saevarma — APPROVED** [review]
Nice job 💯
Just minor comments.

File: libs/portals/admin/ids-admin/src/components/Client/Translations.tsx (hunk 2 above)
Comment (GunnlaugurG) [reply libs/portals/admin/ids-admin/src/components/Client/Translations.tsx:38]: I added it before I added the loader pattern to the request, I forgot to remove it once the loader pattern was implemented :D

**Pull request merged**